core-ui/src/components/Predefined/Create/Deployments/Containers.js

```diff
@@ -0,0 +1,215 @@
+          />
+        )}
+      />
+      <ResourceForm.FormField
+        label={t('deployments.create-modal.simple.docker-image')}
+        value={container.image}
+        setValue={image => {
+          container.image = image;
+          setContainers([...containers]);
+        }}
+        required
+        input={(value, setValue) => (
+          <ResourceForm.Input
+            required
+            value={value}
+            setValue={setValue}
+            placeholder={t(
+              'deployments.create-modal.simple.docker-image-placeholder',
+            )}
+          />
+        )}
+        className="fd-margin-bottom--sm"
+      />
+
+      <ResourceForm.CollapsibleSection
+        title={t('deployments.create-modal.advanced.runtime-profile')}
+        canChangeState={false}
+        defaultOpen
+      >
+        <FormFieldset className="runtime-profile-form">
+          <MemoryInput
+            label={t('deployments.create-modal.advanced.memory-requests')}
+            value={jp.value(container, '$.resources.requests.memory') || ''}
+            setValue={memory => {
+              jp.value(container, '$.resources.requests.memory', memory);
+              setContainers([...containers]);
+            }}
+          />
+          <MemoryInput
+            label={t('deployments.create-modal.advanced.memory-limits')}
+            value={jp.value(container, '$.resources.limits.memory') || ''}
+            setValue={memory => {
+              jp.value(container, '$.resources.limits.memory', memory);
+              setContainers([...containers]);
+            }}
+          />
+        </FormFieldset>
+        <FormFieldset className="runtime-profile-form">
+          <CpuInput
+            label={t('deployments.create-modal.advanced.cpu-requests')}
+            value={jp.value(container, '$.resources.requests.cpu') || ''}
+            setValue={cpu => {
+              jp.value(container, '$.resources.requests.cpu', cpu);
+              setContainers([...containers]);
+            }}
+          />
+          <CpuInput
+            label={t('deployments.create-modal.advanced.cpu-limits')}
+            value={jp.value(container, '$.resources.limits.cpu') || ''}
+            setValue={cpu => {
+              jp.value(container, '$.resources.limits.cpu', cpu);
+              setContainers([...containers]);
+            }}
+          />
+        </FormFieldset>
+      </ResourceForm.CollapsibleSection>
+    </>
+  );
+}
+
+export function Containers({ value: containers, setValue: setContainers }) {
+  const { t } = useTranslation();
+
+  const removeContainer = index => {
+    setContainers(containers.filter((_, i) => index !== i));
+  };
+
+  containers = containers || [];
+
+  if (!containers.length) {
+    return (
+      <MessageStrip type="warning">
+        {t('deployments.create-modal.advanced.one-container-required')}
+      </MessageStrip>
+    );
+  }
+
+  if (containers.length === 1) {
```

Comment:
I thought the idea to not show collapsible containers was when there's no ability to add extra containers (and only the first one is displayed), not when there is only one on the list?